### ISSUE TITLE
External antenna

### ISSUE THREAD

**Issue by jminix** [LilyGO/ESP32-MINI-32-V2.0#4]
I am interested in using the external antenna u.fl connector.  I can not find any information about how to make that work on the pcb.  Can you please provide this information.  A picture showing it would be great.

**Comment by BenODwyer** [comment]
Did you ever work this out? I am also interested in using the external antenna.

**Comment by jminix** [comment]
No, I never figured it out.  Was not a big deal for me though as the signal
ended up being strong enough stock.

**Comment by BenODwyer** [comment]
Ok thanks for the info.

**Comment by kotakomputer** [comment]
You must rotate the Zero Ohm Resistor below PCB Antenna to use External Antenna.

**Comment by ferrets6** [comment]
> You must rotate the Zero Ohm Resistor below PCB Antenna to use External Antenna.

Looking at second pic, you mean I have to rotate resistor from, let's say, position 1-2 to position 1-3 ot to position 3-4?
Thanks!

![IMG_20200407_151959](https://user-images.githubusercontent.com/17033305/78676114-4e518d00-78e6-11ea-8a95-aa5018ec9b99.jpg)
![T7 v1 3 External Antenna](https://user-images.githubusercontent.com/17033305/78676790-34fd1080-78e7-11ea-8bb0-aee88efe75a6.jpg)

**Comment by kyogithub** [comment]
Hi guys, did you finally found how to enable the SMA connector?

**Comment by ferrets6** [comment]
> Hi guys, did you finally found how to enable the SMA connector?

Yes! Position 3-4

**Comment by kyogithub** [comment]
Move element from 1-2 to 3-4 right? or just wire connect 3-4?

**Comment by ferrets6** [comment]
Yes, have to move the 0ohm resistor. 
Or just remove the resistor and wire connect 3-4

**Comment by kyogithub** [comment]
Thanks!!

**Comment by crathje** [comment]
According to the [Schematic.pdf](https://github.com/LilyGO/ESP32-MINI-32-V1.3/blob/master/Schematic.pdf) the mentioned "Zero Ohm" resistor is not a resistor but rather a capacitor (I would go with C10):
![image](https://user-images.githubusercontent.com/619036/142862767-75fb6fbd-f58f-4356-81a3-a6e73b03f103.png)

I would go with the following naming (although I suspect the schematic is a little off):
![image](https://user-images.githubusercontent.com/619036/142865336-7a99cb5e-10dc-4447-8128-8d611b29695a.png)

The R11 seems to be connected to the LNA_IN pin (2) of the ESP:
![image](https://user-images.githubusercontent.com/619036/142866025-c22b0941-9719-4038-a663-0a5e8d3a87b9.png)


Nevertheless it should be ok to rotate the capacitor accordingly in case I am not reading the whole stuff wrong :-)

**Comment by Xylopyrographer** [comment]
Been in conversation with the good folks at LilyGo. They have confirmed the correct way to enable the external antenna. 

Complete summary below for posterity. Note this is for boards marked:
`T7_V1.3`
`20181008` 
on the silkscreen on the upper left below the PCB antenna trace. (Where the USB connector is oriented down.)

Unknown if there are earlier or later versions of the v1.3 board that are different.

Drawing 1 shows the antenna portion of the schematic as given in this repository.
Drawing 2 shows the actual schematic of the antenna portion as per the PCB layout.
Drawing 3 shows how to mod the board to use to the IPEX external antenna connector.
Photo 1 shows the physical layout of all the bits.

I've had one board modified but not yet tested. I'll report back when I've had a chance to compare a stock board against this mod.

Hope this helps.

![1_AntennaSchematic](https://user-images.githubusercontent.com/81269441/173690377-950c401b-83bb-4fe4-8c39-cf2dc32424a5.jpg)
![2_AntennaSchematic-Actual](https://user-images.githubusercontent.com/81269441/173690385-3688ab10-3a36-4e2d-ad24-a232c8f3d605.jpg)
![3_AntennaSchematic-Mod](https://user-images.githubusercontent.com/81269441/173690395-81fa39d6-5dbe-4996-b777-7858fd69a24d.jpg)
![4_BoardPadTraces](https://user-images.githubusercontent.com/81269441/173690400-2cc72ba9-2fe1-4561-b6e4-3a7a79f3ad4e.jpg)

.

**Comment by rkinnett** [comment]
Thanks @Xylopyrographer for relaying that.  How'd your testing go?

What does C10 do here?  I guess it's a high-pass filter?  Is it mainly to protect the ESP32 or does it help clean up RF emissions and/or reception?

I tried to move C10 over to pads 3 and 4 per Lilygo's recommendation but made a mess of it and ended up bridging instead, temporarily.  Not sure if that's advisable long-term.

For reference, Espressif strongly recommends using a pi matching filter, tuned to a specific antenna:  https://www.espressif.com/sites/default/files/documentation/esp32_hardware_design_guidelines_en.pdf  (pg 11) 

![image](https://user-images.githubusercontent.com/25003840/208375940-b94e1835-1ff1-46f6-9abe-5ff9f452f0f7.png)

Someone who knows what they're doing (not me!) could presumably implement a pi filter within the TTGO T7 v1.3 layout by replacing C10 with an inductor and installing appropriate capacitors for C3 and C32 (across pads 1 to 2 and 5 to 6) and bridging pad 1 to 4.
